asmcomp/selectgen.ml

```diff
@@ -142,15 +142,24 @@ let join_array rs =
   let some_res = ref None in
```

Comment:
I'm not really sure that's relevant to the fix, but I shall bear it in mind (the GDB pull request actually changes the meaning of "anonymous", so I need to update the documentation).